dspace/config/dspace.cfg

```diff
@@ -1353,6 +1353,11 @@ websvc.opensearch.formats = html,atom,rss
 # Use -1 to force all bitstream to be served inline
 webui.content_disposition_threshold = 8388608
 
+#### Content Attachment Disposition Formats ####
+#
+# Set which mimetypes, file extensions will NOT be opened inline
+# Files with these mimetypes/extensions will always be downloaded
```

Comment:
I would suggest continuing this comment to say that files with these mimetypes/extensions will always be downloaded regardless of the content disposition threshold.